BookPlayer/Player/PlayerManager.swift

```diff
@@ -373,18 +373,29 @@ class PlayerManager: NSObject {
     }
 
     func stop() {
+        self.audioPlayer?.stop()
+
         guard let book = self.currentBook else {
```

Comment:
if the notification will occur no matter what, let's change this function to the following then:
```
self.audioPlayer?.stop()
var userInfo: [AnyHashable: Any]?

if let book = self.currentBook {
  userInfo = ["book":book]
}

self.currentBooks = []

DispatchQueue.main.async {
            NotificationCenter.default.post(
                name: Notification.Name.AudiobookPlayer.bookStopped,
                object: nil,
                userInfo: userInfo
            )
}
```

Reply:
That's what I was looking for. Much more reasonable.